gbpservice/nfp/base_configurator/api/v1/controllers/controller.py

```diff
@@ -160,13 +143,14 @@ def post(self, **body):
 
             # Assuming config list will have only one element
             config_data = body['config'][0]
-            context = config_data['kwargs']['context']
-            request_info = config_data['kwargs']['request_info']
+            context = body['info']['context']
+            service_type = body['info']['service_type']
+            resource = config_data['resource']
 
-            if 'device_ip' in request_info:
+            if 'device_ip' in context:
```

Comment:
"device_ip" should not be read from 'context' as it is request context, can it be passed and read from 'config['resource'data']'